### PR TITLE
Break out test targets into a separate config

### PR DESCRIPTION
**Pull request by isarkis**
b/354062546

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ services:
     build:
       context: cobalt/docker/linux
       cache_from:
-        - ghcr.io/youtube/cobalt/linux:latest
-    image: ghcr.io/youtube/cobalt/linux:latest
+        - ghcr.io/youtube/cobalt_sandbox/linux:latest
```
[the remark this answers is not in the document]
Comment (oxve) [reply docker-compose.yaml:7]: It is. It was a bug that was introduced earlier that broke docker builds.

File: .github/config/android-arm.json [hunk 2]
```diff
@@ -4,20 +4,22 @@
         "android-arm"
     ],
     "targets": [
+        "ipc_tests",
+        "content_shell",
+        "system_webview_apk",
+        "system_webview_shell_apk",
+        "cobalt:gn_all"
+    ],
+    "test_targets": [
```
Comment (oxve) [inline .github/config/android-arm.json:13]: Change to `unit_test_targets`?

Not super important for this temporary solution but I think it's good practice anyhow.

File: .github/config/linux.json [hunk 3]
```diff
@@ -5,17 +5,19 @@
...
-        "sql_unittests",
-        "url_unittests",
-        "ipc_tests",
-        "mojo_unittests",
-        "gpu_unittests",
-        "gin_unittests",
-        "media_unittests",
-        "content_shell",
-        "cobalt:gn_all"
+      "ipc_tests",
```
[the remark this answers is not in the document]
Comment (oxve) [reply .github/config/linux.json:8]: +1. I think this should be added to the test targets below. It's the same kind of test target as the unittest targets below, despite the name.

**Review by oxve — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaidokert) [reply docker-compose.yaml:7]: We should parameterize this ( later )

File: .github/config/android-arm.json (hunk 2 above)
Comment (isarkis) [reply .github/config/android-arm.json]: done

**Pull request merged**